### ISSUE TITLE
fix snap builds

### ISSUE THREAD

**Issue by sylvestre** [mozilla/sccache#1768]
snap pull can fail like in https://github.com/mozilla/sccache/pull/1764

See the details:

As I say, you can just set it to `false`, since the actual builds from `main` should be coming from https://snapcraft.io/sccache/builds (it needs setting up, if you want that).

If you _want_ the builds from PRs for testing (published to `edge/pr<number>`, you can set the `SNAPCRAFT_TOKEN` secret to the output of:
```shell
snapcraft export-login --snaps sccache --channels="*edge*" --acls package_access,package_push,package_release -`
```

_Originally posted by @Saviq in https://github.com/mozilla/sccache/issues/1764#issuecomment-1549168771_

**Comment by sylvestre** [comment]
@Saviq is that something you could do ? :) thanks

**Comment by Saviq** [comment]
Fixed by #1778

Or, as described, setting the `SNAPCRAFT_TOKEN` secret.
